src/app/dashboard/competitions/[competitionId]/register-form/edit/page.tsx

```diff
@@ -0,0 +1,41 @@
+import { notFound } from "next/navigation";
+
+import { getCompetitionById } from "@/lib/db/competition";
+import { getSignedUrlForReading } from "@/lib/s3";
+import { RegisterFormSettings } from "@/types/competition";
+
+import EditCompetitionRegisterForm from "./components/EditCompetitionRegisterForm";
+
+interface EditRegisterFormPageProps {
+  params: Promise<{
+    competitionId: string;
+  }>;
+}
+
+export default async function EditRegisterFormPage(props: EditRegisterFormPageProps) {
```

Comment:
Y acá también lo cambiaría a:


```suggestion
export default async function EditCompetitionRegisterFormPage(props: EditCompetitionRegisterFormPageProps) {
```